data/mods/Limb_WIP/mutation_eocs/limb_changing_eocs.json

```diff
@@ -0,0 +1,160 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_CHANGING_LIMB_SETUP",
+    "//": "Get the timer started, it should be added to all first-level changing traits",
+    "condition": { "not": { "u_has_var": "limb_setup", "type": "timer", "context": "limbs", "value": "yes" } },
```

Comment:
please stick to math `time` if possible

`{ "math": [ "u_limbs_timer_limb_setup" "=" "time('now')" ] }`